MeshLib/Elements/Element.cpp

```diff
@@ -186,4 +186,20 @@ bool Element::isBoundaryElement() const
         [](MeshLib::Element const*const e){ return e == nullptr; });
 }
 
+#ifndef NDEBUG
+std::ostream& operator<<(std::ostream& os, Element const& e)
+{
+	os << "Element " << e._id << " @ " << &e
```

Comment:
Also, I put the number in the enum, not the string for that type. The latter would require output operator(s) for MeshElemType (and CellType).